src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/helper/builders/CourseActivityBuilder.kt

```diff
@@ -77,6 +80,7 @@ class CourseActivityBuilder(
       ).apply {
         payRates.addAll(this@CourseActivityBuilder.payRates.map { it.build(this) })
         courseSchedules.addAll(this@CourseActivityBuilder.courseSchedules.map { it.build(this) })
+        courseScheduleRules.addAll(this@CourseActivityBuilder.courseScheduleRules.map { it.build(this, this.scheduleStartDate!!) })
```

Comment:
Can you remember why the course activity start date is nullable? I can't 😄

Reply:
Nomis moves in mysterious ways 🤷